styles.css

```diff
@@ -1 +1,70 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+/*mobile grid*/
```

Comment:
De mobile versie wordt niet gevisualiseerd als 1 kolom.  In iedergeval bij mij niet.
![2021-12-20](https://user-images.githubusercontent.com/94354859/146754565-9eb02407-5a17-46d7-a01c-042a59433c42.png)